build.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! List of wellknown SS58 account types as an enum.
+use quote::{format_ident, quote};
+use serde::{self, Deserialize};
+use std::collections::HashMap;
```

Comment:
Please merge the std imports

Reply:
added rustfmt.toml from substrate to fix this.